### ISSUE TITLE
READMEs for each country

### ISSUE THREAD

**Issue by hannahker** [OCHA-DAP/pa-anticipatory-action#148]
Each country within the ```analysis``` directory should have an updated README. [This template](https://github.com/OCHA-DAP/pa-anticipatory-action/blob/develop/analyses/country_template/README.md) can be used to get started. 

We currently have READMEs for: 
- Bangladesh

We need READMEs for: 

- [x] Burkina Faso
- [x] Chad
- [x] Malawi (to be expanded)
- [x] Nepal
- [x] Somalia
- [x] South Sudan
- [x] Ethiopia
- [x] Niger

**Comment by hannahker** [comment]
@Tinkaa and @turnerm I think this makes sense as a first step in improving overall documentation. What are current thoughts on the template that I linked above? I proposed this a while ago so it might be a bit out of date. Curious to hear if you think that any sections are overkill, or if there's anything significant missing. It can also be a super rough starting point with lots of room for flexibility with the different context for each country.

**Comment by turnerm** [comment]
@hannahker thanks so much for organizing this, I think the template looks good, definitely a great starting point as you said. Maybe I will have some comments on it after using it but I think I'll give it a go with Nepal (and probably an update for Bangladesh).

**Comment by Tinkaa** [comment]
Yes thank you Hannah! And sounds good. I added Ethiopia to the todo's as well cause just saw it is in a horrible state. 
I liked @joseepoirier's idea of adding a status tracking to the pilots (Kanban style). 
What do you think about that and @joseepoirier do you think the Readme's would be a good place to incorporate that?

**Comment by joseepoirier** [comment]
I think the Kanban idea came up when Leonardo was suggesting we move away from Trello (per-project cards) to a more status-based way (to do, in progress, done) of tracking our work. I don't know if it's a meaningful distinction, but the Kanban would therefore reflect the work we have to do / are doing / have done, rather than having a Kanban tracking the status of each pilot individually.

If we go that route, we could absolutely add a dated "Current Status" line in the README that indicates whether the pilot's trigger is under development, being monitored, or has activated.

**Comment by Tinkaa** [comment]
Yes I think Kanban was more general but for the readme's I like the idea of a "current status". maybe monitoring vs activated requires too much updating. But at least whether the trigger is currently under development or not

**Comment by hannahker** [comment]
@Tinkaa I agree with you there. What about 'Current status' with options including: 
- under development (for when we're actively working on it)
- completed (once the pilot is finished)
- on hold (pilot is still under development but analytical work is on hold)

**Comment by Tinkaa** [comment]
Would instead of "completed", "implemented" fulfill? as in the pilots are often not really completed even if they have been on monitoring.

**Comment by joseepoirier** [comment]
I agree with Tinka: pilots are never finished, and we don't know which triggers we will revisit after their first version. 

- under development (before a first version of the framework is endorsed)
- endorsed or implemented
- under revision
?

**Comment by Tinkaa** [comment]
I like the endorsed! 
I think on hold would be a good 4th option, e.g. with SSD where we are currently not doing any work but is also not under revision (and mwi flooding is somewhere in between under revision and on hold?)

**Comment by joseepoirier** [comment]
Tinka's suggestion sounds good to me. Good to go from my perspective

**Comment by Tinkaa** [comment]
Perfect. Hannah had the great idea to add batches, so I created one for each status. @hannahker does this sound&look good? 

- under development: https://img.shields.io/badge/STATUS-UNDER%20DEVELOPMENT-%23007CE0
- endorsed: https://img.shields.io/badge/STATUS-ENDORSED-%231EBFB3
- under revision: https://img.shields.io/badge/STATUS-UNDER%20REVISION-%23CCCCCC
- on hold: https://img.shields.io/badge/STATUS-ON%20HOLD-%23007CE0

You can add them to the markdown with  `[![Generic badge](https://img.shields.io/badge/STATUS-ENDORSED-%231EBFB3)](https://shields.io/)`
Resulting in 
[![Generic badge](https://img.shields.io/badge/STATUS-ENDORSED-%231EBFB3)](https://shields.io/)

**Comment by hannahker** [comment]
Thanks @Tinkaa! Just added these to the readme template to keep them visible - see #153

**Comment by hannahker** [comment]
Thanks all for the updates so far! Just checked through and at this point we're close to having this finished with just Chad (@Tinkaa) and Nepal (@turnerm) to be done. No immediate rush, but would be nice to have this finished up soon if possible :)

**Comment by Tinkaa** [comment]
Awesome! @joseepoirier you worked on Chad right? If so, could you do the readme for that?

**Comment by joseepoirier** [comment]
Yes I did. I can write its README. I'll also update BFA's: the trigger's been accepted. 

Should we add a Niger folder, although we do not have code? Our contributions to the trigger were not code-based but we did help get it done. The country is also coloured on the map.

**Comment by Tinkaa** [comment]
Great, thanks! Yes I like the idea of having a folder for each coloured country

**Comment by joseepoirier** [comment]
OK, I'll create a README for Niger.

**Comment by hannahker** [comment]
Just checked through and we're all done here! Thank you 🙏
